components/sendgrid/sendgrid.app.js

```diff
@@ -12,9 +13,17 @@ module.exports = {
     _apiUrl() {
       return "https://api.sendgrid.com/v3";
```

Comment:
As a preference I would suggest having these kind of values in constants. Something like
```javascript
const BASE_URL = "https://api.sendgrid.com";
const VERSION_PATH = "/v3";
```
in that way the maintenance of these would be straightforward in case the api decides to upgrade versions